src/addons/Portal/Portal.js

```diff
@@ -355,7 +355,7 @@ class Portal extends Component {
     document.addEventListener('keydown', this.handleEscape)
 
     const { onMount } = this.props
-    if (onMount) onMount()
+    if (onMount) onMount(undefined, this.props)
```

Comment:
My 0.02 says in that case we pass `null`.  Even with an empty object, the common `e.target.value` would fail.  I think `null` as it says "this is intentionally empty" vs `undefined` which looks like the argument is simply missing.